tests/test_auto_zoom.py

```diff
@@ -0,0 +1,22 @@
+from manim import *
+
+
+def test_zoom():
+
+    s1 = Square()
+    s1.set_x(-10)
+    s2 = Square()
+    s2.set_x(10)
+
+    scene = MovingCameraScene()
+    scene.add(s1, s2)
+    scene.play(scene.camera.auto_zoom([s1, s2]))
```

Comment:
We should make sure that nothing is actually rendered.

```suggestion
    with tempconfig({'dry_run': True}):
        scene = MovingCameraScene()
        scene.add(s1, s2)
        scene.play(scene.camera.auto_zoom([s1, s2]))
```